pkg/cri/server/service_test.go

```diff
@@ -86,3 +92,16 @@ func TestLoadBaseOCISpec(t *testing.T) {
 	assert.Equal(t, "1.0.2", out.Version)
 	assert.Equal(t, "default", out.Hostname)
 }
+
+func TestAlphaCRIWarning(t *testing.T) {
+	ctx := context.Background()
+	ws := servertesting.NewFakeWarningService()
+	c := instrument.NewAlphaService(newTestCRIService(), ws)
+
+	c.ListImages(ctx, &v1alpha2.ListImagesRequest{})
```

Comment:
This part is necessary to trigger the warning because it won't be emitted until an actual call is made over v1alpha2?

Reply:
yes. By default the cri service will register both v1 and v1alpha2 services. So I think it would be more accurate to only emit the warning when a v1alpha2 api is called.